### DISCUSSION TITLE
Update p384 crate version to include "p384: remove unused fiat-crypto variables" commit

### DISCUSSION THREAD

**Discussion by Yashinde145** [RustCrypto/elliptic-curves#1102] [General]
https://github.com/RustCrypto/elliptic-curves/commit/7652c5842cde60acc1e739ebbba34614cb158f8b fixes https://github.com/rust-lang/rust/issues/122357. 

Also, there were other reports of similar error:
https://github.com/rust-lang/rust/issues/121814
https://github.com/rust-lang/rust/issues/113612
https://github.com/rust-lang/rust/issues/104021

This error was there for a while and blocked many of the p384 rust debug builds applications.
The fix was merged into the pre-release versions "p384/v0.14.0-pre.2" and "p384/v0.14.0-pre.1."
and current stable version of the RustCrypto p384 crate is v0.13.0, which is also the default version for building with Rust. 

This is a significant fix and it would be nice to have it in stable version of p384 crate.
Can we have a new stable version update of p384 crate along with this fix ?
Or 
Can this fix can be backported to "v0.13.0" of p384 crate.

Please let me know your views on it.

CC: @tarcieri 

Thanks.

**Comment by Yashinde145** [comment]
Hi,
any feedback on the suggestions above?

Edit: @tarcieri, a gentle ping.
Can you please let us know if the crate stable version can be updated along with the above fix anytime soon?
This would be very helpful as it fixes a significant issue causing segmentation fault.
Thanks.

**Comment by tarcieri** [comment]
There is a huge amount of work to be done before another stable release.

You can track our upstream traits here. We'll need these all with a newly versioned stable release of several of these first before we can even start getting to the elliptic curve crates: https://github.com/RustCrypto/traits/issues/1571

I would also like to rip out the existing transformed fiat-crypto backends and go back to something that looks close to the verbatim output now that `const_mut_refs` are stable. I also plan on adding backends based on `crypto-bigint`'s modular arithmetic.

**Reply by Yashinde145** [reply]
Thank you for the insights into the effort required for the next stable release.

Given the requirements and dependencies, it seems like it might take some time to get the new release. Is it possible to backport the fix from  https://github.com/RustCrypto/elliptic-curves/commit/7652c5842cde60acc1e739ebbba34614cb158f8b to the current stable version (v0.13.0)?

Also, do you have any tentative timeline for when the new release might be expected?

**Reply by tarcieri** [reply]
Yes, I can publish a backport.

I don't have a timeline on releasing new stable versions of these crates, again due to the massive amount of work involved and the fact we're volunteers doing this in our free time, and also are dependent on downstream dependencies which also need to be updated.

**Reply by Yashinde145** [reply]
Would it be advisable for us to raise a PR and backport the fix on your behalf?

**Reply by tarcieri** [reply]
It's released

**Reply by Yashinde145** [reply]
Will the p384 v0.13.1 be the part of upcoming rust v1.85 vendor sources releasing next week(Feb 20)?

**Reply by tarcieri** [reply]
huh?
